tools/federation/src/main/java/org/eclipse/rdf4j/federated/algebra/FedXDescribeOperator.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.federated.algebra;
+
+import org.eclipse.rdf4j.federated.structures.QueryInfo;
+import org.eclipse.rdf4j.query.algebra.DescribeOperator;
+import org.eclipse.rdf4j.query.algebra.TupleExpr;
+
+/**
+ * Specialized {@link DescribeOperator} Node for maintining {@link QueryInfo}.
+ * 
+ * @author Andreas Schwarte
+ *
+ */
+public class FedXDescribeOperator extends DescribeOperator implements QueryRef {
```

Comment:
Can we call this `FederatedDescribeOperator` so it's a bit more generic?

Reply:
Sure, I have applied this now